### ISSUE TITLE
Feature Request: Translate Below

### ISSUE THREAD

**Issue by Jxck** [soerenuhrbach/vscode-deepl#17]
Hi Team

This extension is so nice and I'm big fan of it !! Thanks so much 👍
My request is adding a command for "Translate Below".

## Motivation

When translating some documents into other lang. I need to check that sentences are translated correctly.
But current extension overwrites original sentences into translated one.

for example

```
# before
Hello

# after
Bonjour
```

In this case, I can't check the original sentences after translate.

## Request

Solve this problem is so easy, just adding translated result below the original line.

```
# before
Hello

# after
Hello
Bonjor
```

It makes so easy for me to check/compare the result and if I don't need original line, I'll remove it manually. (just ctl+k)

## Contribution

I thinks this change is just adding feature with no breaking changes.
If you feel positive for this change, I can send PR for here.

Any comments are welcome !

Thanks
Jack

**Comment by soerenuhrbach** [comment]
Hi Jack,

I am glad that you like the extension. 

Your idea sounds pretty good to me.  Feel free to contribute if you want to!

Sören

**Comment by soerenuhrbach** [comment]
I will release a new version with your new command soon.

**Comment by Jxck** [comment]
Thanks!

**Comment by Jxck** [comment]
@soerenuhrbach when will it released ?

**Comment by Jxck** [comment]
@soerenuhrbach reminder

**Comment by soerenuhrbach** [comment]
Hi @Jxck,

Sorry. I will release it tomorrow!

**Comment by soerenuhrbach** [comment]
Today was "tomorrow".. just released the new command with version 1.0.5.
